lib/control-app.js

```diff
@@ -5,6 +5,10 @@ var express = require('express');
 var expstate = require('express-state');
 var browserify = require('browserify');
 var im = require('istanbul-middleware');
+var watchify = require('watchify');
+var _ = require('lodash');
```

Comment:
In fact, I had used lodash specific builds before and stopped because in the end, npm install grabs MORE things, since the 'lodash' package embed everything, while specific builds have a lot of dependencies.

In the end, you might endup with a bigger size of your installed npm package.

What we can do (for code reading) is this: `var assign = require('lodash/object/assign')` which I like a lot. In browserify bundles it will only embed required code (but we are in node here).